src/mame/tvgames/spg2xx.cpp

```diff
@@ -2568,6 +2616,19 @@ CONS( 2008, ddr33v,     0,        0, spg2xx,    ddr33v,    spg2xx_game_ddr33v_st
 // PCB has 'Anpanman TV 2006 Ver 1.4' printed on it, ROM has SPG260 header.  Uses custom built-in keyboard, no display built into the unit.
 CONS( 2006, anpantv,    0,        0, spg2xx,    spg2xx,    spg2xx_game_state,          empty_init,    "Bandai",                                                "Anpanman TV (Japan)",                                                   MACHINE_NOT_WORKING )
 
+// Has an AT24C08, not currently hooked up (probably for storing database unlocks)
+// 
+// There is also a card reader/scanner which can read barcodes from Digimon cards
+// and IR connectivity which allowed for data exchange with various services using
+// an external device, including transfering characters to/from an arcade game.
+// Neither is currently emulated
+// 
+// Will report 'ERROR' sometimes, maybe as a result of these not being hooked up.
+CONS( 2006, dmbtjunc,   0,        0, spg2xx,    dmbtjunc,  spg2xx_game_state,          empty_init,    "Bandai",                                                "Let's! TV Play Digital Monster Battle Junction (Japan)",                MACHINE_NOT_WORKING )
+
+// 川島隆太教授監修 脳と体を鍛える 体感頭脳ファミリーマットレ  - Let's! TV Play branding appears on the box
+CONS( 2006, ban_krkk,   0,        0, spg2xx,    ban_krkk,  spg2xx_game_state,          init_crc,      "Bandai",                                                "Let's! TV Play Kawashima Ryuuta Kyouju Kanshuu - Nou to Karata o Kitaeru Taikan Zunou Family Mattore (Japan)", MACHINE_IMPERFECT_SOUND )
```

Comment:
```suggestion
CONS( 2006, ban_krkk,   0,        0, spg2xx,    ban_krkk,  spg2xx_game_state,          init_crc,      "Bandai",                                                "Let's! TV Play Kawashima Ryuuta Kyouju Kanshuu - Nou to Karada o Kitaeru Taikan Zunou Family Mattore (Japan)", MACHINE_IMPERFECT_SOUND )
```

Same for the PR description